crates/flux-middle/src/rty/mod.rs

```diff
@@ -119,6 +119,10 @@ impl AdtSortVariant {
         self.sorts.fold_with(&mut SortSubst::new(args))
     }
 
+    pub fn field_sorts_id(&self) -> List<Sort> {
```

Comment:
maybe

```suggestion
    pub fn field_sorts_instantiate_identity(&self) -> List<Sort> {
        self.sorts.clone()
    }
```

or `field_sorts_identity`